site/content/docs/5.0/components/dropdowns.md

```diff
@@ -260,7 +260,7 @@ Button dropdowns work with buttons of all sizes, including default and split dro
         <li><a class="dropdown-item" href="#">Separated link</a></li>
       </ul>
     </div><!-- /btn-group -->
-    <div class="btn-group ml-2">
+    <div class="btn-group ml-sm-2">
```

Comment:
This now no longer matches L290, where we still use `ml-2`. I'd suggest deleting both `ml-2` instances instead.

Reply:
@mdo set `L263` and `L290` the same with deleting both `ml-2`.
I'd adjusted margin of `> .btn-toolbar > .btn-group,` in the previous commit and I realized that it affected button group docs. So, I additionally adjusted margin of `> .btn-toolbar > .input-group` to set proper margin in button group docs